default/techs.txt

```diff
@@ -2751,6 +2753,113 @@ Tech
     unlock = Item type = ShipPart name = "SP_BIOTERM"
     graphic = "icons/ship_parts/bioterm.png"
 
+Tech
+    name = "SHP_EMP"
+    description = "SHP_EMP_DESC"
+    short_description = "SHIP_PART_UNLOCK_SHORT_DESC"
+    techtype = Application
+    category = "SHIPS_CATEGORY"
+    researchcost = 450 * [[TECH_COST_MULTIPLIER]]
+    researchturns = 3
+    prerequisites = [
+        "SHP_BOMBARD"
+        "LRN_FORCE_FIELD"
+    ]
+    unlock = Item type = ShipPart name = "SP_EMP"
+    graphic = "icons/ship_parts/death-spore.png"
+
+Tech
+    name = "SHP_EMO"
+    description = "SHP_EMO_DESC"
+    short_description = "SHIP_PART_UNLOCK_SHORT_DESC"
+    techtype = Application
+    category = "SHIPS_CATEGORY"
+    researchcost = 75 * [[TECH_COST_MULTIPLIER]]
+    researchturns = 5
+    prerequisites = [
+	"SHP_EMP"
+	"SHP_FRC_ENRG_COMP"
```

Comment:
Code formatting nitpick: the above two lines are indented by one tab, pls replace with 8 whitespaces instead.